scripts/optimistic-tock.js

```diff
@@ -0,0 +1,79 @@
+const holidays = require("@18f/us-federal-holidays");
+const moment = require("moment-timezone");
+const scheduler = require("node-schedule");
+const utils = require("../utils");
+
+const TOCK_API_URL = process.env.HUBOT_TOCK_API;
+const TOCK_TOKEN = process.env.HUBOT_TOCK_TOKEN;
+
+let util;
+
+let reminder = (robot) => {
+  const message = {
+    username: "Happy Tock",
+    icon_emoji: "happytock",
+    text: "Don't forget to <https://tock.18f.gov|Tock your time>!",
+    as_user: false,
+  };
+
+  reminder = (tz) => async () => {
+    // Get the Slack users in this timezone.
+    const tzUsers = (await util.getSlackUsers()).filter((u) => u.tz === tz);
+
+    // Now get all the folks who have not submitted their current Tock.
+    const truants = await util.tock.get18FTockTruants(moment.tz(tz), 0);
+
+    const tockSlackUsers = (await util.tock.get18FTockSlackUsers())
```

Comment:
it seems a little funny that we have to `getSlackUsers` then `get18FTockSlackUsers`; is the only extra information that `getSlackUsers` is returning is the timezone information? 

would it make sense to modify `get18FTockSlackUsers` to return that bit of data as well, and then use it in line 21 so we don't have to make two api calls?

Reply:
Ooh, smart! That's way easier than unioning two arrays, too.

Reply:
i think you can remove the `getSlackUsers` export from `utils.js` as well with this change! (unless you like having it exported for the tests?)

Reply:
Good catch! I'm leaning towards leaving it exported because of the tests, even though the functionality is already covered by other tests, just because it'd feel weird to delete some tests here that might be need to be put back in the future, if that method got exposed separately later. But no premature optimization! :ohno: